### ISSUE TITLE
-bt (time stations) option currently doesn't show any results

### ISSUE THREAD

**Issue by cobaltpet** [cobaltpet/swl-tool#29]
At some point in the past day or two, the -bt option stopped working. I can do

`-b WWV`

but:

```
$ ./swl-tool.rb -bt    
swl-tool.rb version 2017-09-06 0014UTC by Eric Weatherall : cobaltpetxxxxxxxcom : http://cobaltpet.blogspot.com/
credit: Shortwave broadcast schedule data from EiBi
info: Loaded 12377 schedule entries

```

**Comment by cobaltpet** [comment]
The problem was introduced with the fix for #26 -- BroadcastEntry no longer stores the raw language code from the EiBi data, it now stores the human-readable string.

**Comment by cobaltpet** [comment]
Fix committed
